pycbc/workflow/core.py

```diff
@@ -198,7 +198,10 @@ def __init__(self, cp, name,
             raise TypeError("Failed to find %s executable " 
                             "at %s" % (name, exe_path))
         
-        self.add_pfn(exe_path)
+        if exe_path.startswith('gsiftp://'):
+            self.add_pfn(exe_path,site='pycbc-code')
```

Comment:
I think we've used the convention of using site='nonlocal' for this kind of use.   
Also, a small thing. This needs a space after the comma.